src/ripple/app/ledger/impl/DisputedTx.h

```diff
@@ -61,9 +62,7 @@ class DisputedTx
         return mOurVote;
     }
 
-    // VFALCO TODO make this const
-    // VFALCO TODO Don't return a Serializer (doh)
-    Serializer& peekTransaction ()
+    Tx_t const& peekTransaction () const
```

Comment:
I would have gone with `tx` instead of `peekTransaction`. We're not really peeking anymore, its well defined that a disputed tx holds a tx in it.